src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -211,6 +213,16 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
     startBlock: number,
     endBlock: number
   ): Promise<TransactionReceipt> {
+    // Do not submit batch if gas price above threshold
+    const gasPriceInGwei = parseInt(
+      ethers.utils.formatUnits(await this.signer.getGasPrice(), 'gwei'), 10
+    )
+    if (gasPriceInGwei > this.gasThresholdInGwei) {
+      this.log.info(`Current gas price ${gasPriceInGwei} is higher 
+                      than gas price threshold ${this.gasThresholdInGwei}`)
```

Comment:
This is going to result in a `/n` in the logline itself. There are two ways to fix it - concatenate the two strings with `+` or put the newline directly after `this.log.info(`

Reply:
ooo completely forgot abt this, fixing